src/strings/nfa-strings.cc

```diff
@@ -144,6 +144,17 @@ void ShortestWordsMap::update_current_words(LengthWordsPair& act, const LengthWo
     act.first = dst.first + 1;
 }
 
+std::set<mata::Symbol> mata::strings::get_one_symbol_words(const Nfa& nfa) {
+    std::set<mata::Symbol> one_symbol_words;
+    for (State init : nfa.initial) {
+        for (const auto& move_from_init : nfa.delta[init]) {
```

Comment:
No longer a `move`. It should be renamed to `symbol_post_from_init` or similar. Additionally, if possible, could you replace the `auto` here with `SymbolPost` for greater clarity?

Reply:
done